src/locales/en/translation.json

```diff
@@ -453,13 +453,19 @@
     }
   },
   "transaction": {
+    "delegate": "Delegate",
+    "deposit": "Deposit",
     "header": "Transaction",
-    "warningMultipleTransactionsSameHash": "Please make sure you're looking at the right transaction. There is more than one transaction with this hash, which is extremely rare. We are showing the most recent successful one.",
     "rawData": "Raw Data",
+    "started": "Started",
+    "startedDescription": "{{ method }} has started, final result will be known in the next block",
```

Comment:
"Undelegate has started" is a misleading combination

Reply:
What label would be more clear for you exactly?